docs/reference/modules/discovery/master-election.asciidoc

```diff
@@ -0,0 +1,48 @@
+[[master-election]]
+=== Master Election
+
+Elasticsearch uses an election process to agree on an elected master node, both
+at startup and if the existing elected master fails. Any master-eligible node
+can start an election, and normally the first election that takes place will
+succeed. Elections only usually fail when two nodes both happen to start their
+elections at about the same time, so elections are scheduled randomly on each
+node to avoid this happening. Nodes will retry elections until a master is
+elected, backing off on failure, so that eventually an election will succeed
+(with arbitrarily high probability). The following settings control the
```

Comment:
I would recommend moving/copying the high-level overview of elections to a higher-level topic and then leaving the rest as a simple settings reference page.

Reply:
Moved to quorums.asciidoc for now in 64564c0fb2a. I think we might promote some of that material as you suggest, but not in this PR.